lib/luadata.c

```diff
@@ -163,30 +177,46 @@ lunatik_object_t *luadata_new(void *ptr, size_t size, bool sleep)
 		data->ptr = ptr;
 		data->size = size;
 		data->free = false;
+		data->editable = editable;
 	}
 	return object;
 }
 EXPORT_SYMBOL(luadata_new);
 
-int luadata_reset(lunatik_object_t *object, void *ptr, size_t size)
+#define luadata_resetter(L, p, sz)		\
```

Comment:
why can't it be an inline?

Reply:
I need to have `data->editable` after the macro to set it here : https://github.com/luainkernel/lunatik/pull/139/files#diff-7af6ccd3b938d3efe6965a52e59f84c7f34aae6f825864536661cf83f544a277R199-R200

Reply:
I still miss the point.. AFAIK, you can do the same in a inline function.. perhaps I'm missing something.. the same accounts for the checker, for me